.github/ISSUE_TEMPLATE/page-report-es.yml

```
@@ -5,58 +5,58 @@ body:
   - type: markdown
     attributes:
       value: |
-        ### Before you start
+        ### Antes de comenzar
 
-        **Want to change this page yourself?** This content is open source!
-        ↩ Go back and use the _Edit on GitHub_ link on the page.
+        **¿Quiere cambiar esta página por usted mismo?** ¡Este contenido es de código abierto!
+        ↩ Regrese y use el enlace _Editar en GitHub_ en la página.
 
-        **Is your issue about the browser compatibility table?**
-        ↩ Go back and use the _Report problems with this compatibility data on GitHub_ link on the page.
+        **¿El problema es acerca de la tabla de compatibilidad con los navegadores?**
+        ↩ Regrese y use el enlace _Reportar problemas con los datos de compatibilidad en GitHub_ en la página.
 
-        **Need help with a browser?**
-        🙋 To get help with [Firefox](https://support.mozilla.org/en-US/kb/file-bug-report-or-feature-request-mozilla), [Chrome](https://support.google.com/chrome/answer/95315?hl=en-GB&ref_topic=7439544), [Safari](https://www.apple.com/feedback/safari.html), or another browser, check the browser's support site.
+        **¿Necesita ayuda con su navegador?**
+        🙋 Para obtener ayuda con [Firefox](https://support.mozilla.org/en-US/kb/file-bug-report-or-feature-request-mozilla), [Chrome](https://support.google.com/chrome/answer/95315?hl=en-GB&ref_topic=7439544), [Safari](https://www.apple.com/feedback/safari.html), o algún otro navegador, visite la página de soporte del navegador.
 
         ---
   - type: input
     id: mdn-url
     attributes:
       label: MDN URL
-      description: Set automatically. Do not modify.
+      description: Se establece automáticamente. No la modifique.
```

Para usar el infinitivo preferente

```suggestion
      description: Se establece automáticamente. No modificar.
```